### DISCUSSION TITLE
打开启动器后偶尔会出现没有 游戏设置 和 选择版本

### DISCUSSION THREAD

**Discussion by XLX9200** [Meloong-Git/PCL#2380] [提问]
![image](https://github.com/Hex-Dragon/PCL2/assets/140784891/2ecaa1d9-ad00-4f84-8320-f8fc374bb856)

**Comment by WForst-Breeze** [comment]
按一下 f12？

**Comment by XiaoLLiu617** [comment]
卡顿情况？？？，建议右键刷新（可能是窗口渲染问题，具体要看其他程序有没有一样的卡顿）
